Add data samples and requirements to Accessibility features

Sergiodero opened the pull request:
Changes to Accessibility features include:

- Adds data samples for each feature
- Adds requirements for each feature
- Fixes link to reference.md

Sergiodero committed:
Add data samples and requirements

Changes include:

- Adds data samples for each feature
- Adds requirements for each feature
- Fixes link to reference.md

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
 
 Wheelchair accessibility fields make it possible to indicate if a stop and vehicle can accommodate users using wheelchairs, allowing them to plan their trips based on the most convenient option for their needs.
 
+**Pre-requirements**: Implement Base features, if provding wheelchair accesibility for station entrances and exits Location Types features is also required.
```
This sentence isn't super easy to understand, do we need to specify "if provding wheelchair accesibility..." since we're already inside "wheelchair accessibility"?
Also, there is a small typo to acces**s**ibility.

Reccomendation
**Pre-requirements**: 
- Base features (with the link to it)
- Location Types (with the link to it)

Sergiodero replied:
I adjusted the format to match with your suggestion but kept the note as "(if providing information for station entrances and/or exits)

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
 
 Wheelchair accessibility fields make it possible to indicate if a stop and vehicle can accommodate users using wheelchairs, allowing them to plan their trips based on the most convenient option for their needs.
```
I suggest adding "If you are showing wheelchair accessibility information, it is important to differentiate between "no access" and "no information", as knowing a stop is not accessible is as important as knowing it is.

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following tables indicate that in stop `TAS001` some vehicles can be boarded by a rider in a wheelchair `wheelchair_boarding=1` in `stops.txt` while using `trips.txt` it is specified that the vehicle providing service for trip `AWE1` can accommodate at least one rider in a wheelchair, while the vehicle used for trip `AWE2` cannot. 
```
This is a bit of a long sentence, there might be a better way to explain that both the trip and the stop need to be wheelchair accessible. 

Here is an alternative suggestion:

```suggestion
    The following sample shows wheelchair boarding is available at stop `TAS001` with  `wheelchair_boarding=1`.
The vehicle used in trip `AWE1` is equipped to accommodate at least one wheelchair, and the vehicle used in trip `AWE2` is not.
Both the stop and trip must be wheelchair accessible for a passenger to be able to access a trip at the given stop.

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`wheelchair_boarding` |
+|[trips.txt](/documentation/schedule/reference/#tripstxt)|`wheelchair_accessible`|
+
+??? note "Sample Data"
```
Need a line break

```suggestion
??? note "Sample Data"

```

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
 
 Text-to-speech allows to provide the necessary inputs to convert text into audio, ensures that riders using assistive technology to read text aloud are getting the right stop names when using the transit service.
 
+**Pre-requirements**: Implement Base features.
```
I recommend to simply have "Base features" with a link to it and have "Pre-requirement" without the "s" when there is only one.

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
-|[stops.txt](/schedule/reference/#stopstxt)|`tts_stop_name` |
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`tts_stop_name` |
+
+??? note "Sample Data"
```
```suggestion
??? note "Sample Data"

```

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table assigns an orange color to route RA using the HEX color code D95700, while also specifying that text should be rendered black.
```
```suggestion
    The following sample assigns an orange color to route RA using the HEX color code `D95700`, while also specifying that text should be rendered black using HEX color code `0`.

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table specifies that the vehicle used in trip `AWE1` is able to accommodate at least one bicycle on board, while also specifying that bicycles cannot be accommodated in the vehicle used for trip `AWE2`.
```
I am probably missing nuances of what "while" means in English here, but for me it implies a duration (it could be replaced by "during this time,). Also, having the first portion of the sentence with "the vehicle" as a subject and the second part with "the bike" as the subject can imply these are two different things.

isabelle-dr replied:
Here is a suggestion to make this sentence simpler:
```suggestion
 The following sample specifies that the vehicle used in trip `AWE1` can accommodate at least one bicycle on board, and the vehicle used in trip `AWE2` cannot.
```

isabelle-dr commented on docs/getting_started/features/accesibility.md:
```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The first table specify the headsigns to be used by trips `AWE1` and `AWE2`, while the second one indicates that `AWE1`’s headsign will be modified after stop `TAS004` overriding the one specified in `trips.txt`.
```
Same comment for using "while"

isabelle-dr reviewed (changes requested):
Hello @Sergiodero , thank you for working on this! 

I made a few suggestions to make the paragraphs less verbose

Some meta comments:
- We need a line break after the _??? note "Sample Data"_ in order for it to render properly
- It would be nice to replace "the following table" with "the following sample" to be consistent
- In the Pre-requirements:
  - I think we should have a list (with links), not a sentence: I'd remove the "implement" and just name the feature
  - I would remove the "s" if there is only one feature
- I would avoid the word "while" when there is no notion of duration and stick to "and".

Sergiodero committed:
Update sample description for wheelchair accessibility

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Adds line break on line 16

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Adds line break on line 54

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Updates sample description for route colors

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Updates sample description for bike allowed

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Address first round of comments

Meta and specific comments are addressed

Sergiodero commented:
Comments are now addressed, I'm anticipating that I might change the order of the information (moving the pre-requirements below, after the table and before the data samples), but I'll do that for all features in a separate PR.

Sergiodero committed:
Remove S if only one requirement

isabelle-dr reviewed (approved):
Thank you for making the changes! Approving this PR, but leaving two additional comments on this one. 
**First**: It took me some back-and-forth with the spec to understand why there is a dependency with _Location Types_ for the _Wheelchair Accessibility_ feature. Few reasons:
1. reading the spec, it seems like it's also possible to specify accessibility to platforms (or boarding areas), so maybe we need to change the:

> (if providing information for station entrances and/or exits)

To (I think we can remove the parenthesis):

> when defining accessibility information about station locations such as entrances/exists or boarding areas.

2. `location_type` doesn't appear in the example. I know this specific example doesn't need it, but I think adding it, even if left empty, would make folks that aren't super familiar think "ah right, it's that field in stops.txt used to model stations. I could add a value to it and then the `wheelchair_boarding` field would apply to this location". 

**Second**: there is a typo in "accessibility" in the file name, and in the .yml file: we're missing the "s".

<img width="564" alt="Screenshot 2024-04-25 at 11 11 37 AM" src="https://github.com/MobilityData/gtfs.org/assets/63653518/c3d1dfb5-ebc7-4683-aed2-990c258043b9">

Sergiodero committed:
Address outstanding comments

- Fixes typo in navigation for Accessibility
- Adds location type in sample
- Modifies note for location type feature requirement
